modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js

```diff
@@ -12,73 +12,33 @@
  * details.
  */
 
-import {ItemSelectorDialog, PortletBase, openModal} from 'frontend-js-web';
+import {
+	DefaultEventHandler,
+	ItemSelectorDialog,
+	openModal,
+} from 'frontend-js-web';
 import {Config} from 'metal-state';
 
 /**
- * @class FragmentCollectionsView
+ * @class FragmentCollectionsViewDefaultEventHandler
  */
-class FragmentCollectionsView extends PortletBase {
-
-	/**
-	 * @inheritdoc
-	 * @review
-	 */
-	created() {
-		this._fragmentCollectionsFm =
-			document[this.ns('fragmentCollectionsFm')];
-		this._handleComponentReady = this._handleComponentReady.bind(this);
-
-		Liferay.componentReady(this.ns('actionsComponent')).then(
-			this._handleComponentReady
-		);
-
-		Liferay.componentReady(this.ns('emptyResultMessageComponent')).then(
-			this._handleComponentReady
-		);
-	}
-
-	/**
-	 * Handles a componentReady event and adds a click handler to the
-	 * given fragment.
-	 * @param {{ on: function }} component
-	 * @private
-	 * @review
-	 */
-	_handleComponentReady(component) {
-		const ACTIONS = {
-			deleteCollections: this._deleteCollections,
-			exportCollections: this._exportCollections,
-			openImportView: this._openImportView,
-		};
-
-		component.on(['click', 'itemClicked'], (event, facade) => {
-			let itemData;
-
-			if (event.data && event.data.item) {
-				itemData = event.data.item.data;
-			}
-			else if (!event.data && facade && facade.target) {
-				itemData = facade.target.data;
-			}
-
-			if (itemData && itemData.action && ACTIONS[itemData.action]) {
-				ACTIONS[itemData.action].call(this);
-			}
-		});
-	}
+class FragmentCollectionsViewDefaultEventHandler extends DefaultEventHandler {
 
 	/**
 	 * Opens an item selector to remove selected collections
 	 * @private
 	 * @review
 	 */
-	_deleteCollections() {
+	deleteCollections() {
 		this._openFragmentCollectionsItemSelector(
 			Liferay.Language.get('delete'),
 			Liferay.Language.get('delete-collection'),
 			this.viewDeleteFragmentCollectionsURL,
 			(selectedItems) => {
+				const fragmentCollectionsFm = document.getElementById(
```

Comment:
Another `Fm` to kill. I won't comment on any others, so as to avoid spamming people subscribed to this PR; but if you want to avoid the abbreviations, would be worth doing a sweep through.

Reply:
Done, sorry... just moving code around, missed all of these...